### PR TITLE
feat: CRN-652 add tag selector to the Shared Outputs form

### PR DESCRIPTION
**Pull request by marot**

### CODE REVIEW

File: packages/react-components/src/organisms/TeamCreateOutputExtraInformationCard.tsx [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+import { FormCard, LabeledMultiSelect } from '../molecules';
+import { noop } from '../utils';
+
+type TeamCreateOutputExtraInformationProps = {
+  values: string[];
+  onChange?: (values: string[]) => void;
+};
+
+// The list is auto generated // TODO: Where should we put this?
+const tags = [
```
Comment (jhonny-m) [inline packages/react-components/src/organisms/TeamCreateOutputExtraInformationCard.tsx:11]: we already have these in the code at `expertiseAndResourceSuggestions`

Comment (marot) [reply packages/react-components/src/organisms/TeamCreateOutputExtraInformationCard.tsx:11]: Ah thank you - I didn't think of searching for them.

The list seems to be updated:
- ~300 new items
- There are many renames
- Some deletions

I'd propose to reuse them and update them in a separate issue.

Comment (marot) [reply packages/react-components/src/organisms/TeamCreateOutputExtraInformationCard.tsx:11]: Follow up: The list should be separate. I will move it next to `expertiseAndResourceSuggestions` with a note that the duplication is intentional

File: apps/storybook/src/FormCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { FormCard } from '@asap-hub/react-components';
+
+export default {
+  title: 'Molecules / FormCard',
```
Comment (kmaid) [inline apps/storybook/src/FormCard.stories.tsx:4]:
```suggestion
  title: 'Molecules / Form Card',
```

File: apps/storybook/src/FormCard.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  component: FormCard,
+};
+
+export const Normal = () => (
+  <FormCard title={'What are you sharing?'}>Card content</FormCard>
```
Comment (kmaid) [inline apps/storybook/src/FormCard.stories.tsx:9]: Use the text Knob so designers can play around with this value

**Review by kmaid — APPROVED** [review]

**Pull request merged**